web/client/plugins/Login.jsx

```diff
@@ -30,13 +30,27 @@ import { isAdminUserSelector } from '../selectors/security';
   * - `url` optional URL to redirect in case of `openID`. By default it will use the standard convention to `rest/geostore/{provider}/login`.
   * - `imageURL` optional URL for the image to use in the link of the login form. (certain pre-defined services like `google` may have their own default logo)
   * - `title` a text to show in the link to the login page of the provider, if logo is present, this text is used as `alt` text in for the image.
-  * Example of configuration.
+ * Example of configuration.
```

Comment:
Check your PR to prevent lines committed in error, like this.
```suggestion
  * Example of configuration.
```